lua/hbac/init.lua

```diff
@@ -92,7 +92,38 @@ local function close_unused()
 	end
 end
 
+
+-- Create commands to add and remove buffers from the persistant_buffers table
+M.release_buffer = function(bufnr)
+	bufnr = bufnr or vim.api.nvim_get_current_buf()
+	M.persistant_buffers[bufnr] = nil
+end
+
+M.commands = {
+	pin = M.persist_buffer,
+	unpin = M.release_buffer,
```

Comment:
```suggestion
	pin = M.pin_buffer,
	unpin = M.unpin_buffer,
```
let's use the pin metaphor cause why not